packages/visualizations/src/components/Chart/Chart.svelte

```diff
@@ -69,14 +73,15 @@
     $: {
         // Reactively update chart configuration
         const chartOptions = chartConfig.options || {};
+        const legend = options.legend?.custom ? { display: false } : buildLegend(options);
```

Comment:
Just a detail, but maybe a small comment here or a FIXME as it's not easy to understand why there is a `display: false` and why we are doing the split of the legend this way ?